recyclerview-fastscroll/src/main/java/com/simplecityapps/recyclerview_fastscroll/views/FastScrollRecyclerView.java

```diff
@@ -369,14 +374,30 @@ private void getCurScrollState(ScrollPositionState stateOut) {
         }
 
         View child = getChildAt(0);
-
         stateOut.rowIndex = getChildAdapterPosition(child);
+
+        // don't do anything if we have a header item
+        if (getLayoutManager().getChildAt(0).getHeight() < getLayoutManager().getChildAt(1).getHeight()) { // identify header
+            stateOut.rowHeight = lastRowHeight;
+            stateOut.rowIndex = lastRowIndex;
+            stateOut.rowTopOffset = lastTopOffset;
+            return;
```

Comment:
Since there will not be executed if there is no content in the photo view, and __if there is any item__, then we can guarantee that there will be a child at 0 and a child at one since we always have one title and multiple items. Thus, I included an __if check__ here to see if they identify the headers and if it is a header, the state of recycler view will remain the same. This limited the jump to minimum.